### PR TITLE
Improve readability for list of blog posts

### PR DESCRIPTION
**Pull request by metalmatze**
The list of blog posts isn't that wide anymore, improved readability.
Also I removed disqus from the list of blog posts.

> I removed disqus from this list view of blog posts because I don't see a
real value in having comments there. They should be made to the specific
blog post, otherwise go on gitter.

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -48,12 +48,6 @@
 				</div>
 			{{ end }}
 		</div>
-
-		{{ if .Site.DisqusShortname }}
```
Comment (tboerger) [inline layouts/index.html]: We got this on every page except on the landingpage.

**Comment by tboerger** [comment]
Please keep the formatting of the markdown content.

**Comment by andreynering** [comment]
LGTM

**Comment by metalmatze** [comment]
Revert all changes and only applied the stuff agreed on. Please LGTM and squash mergen. 👍

**Comment by tboerger** [comment]
LGTM

**Pull request merged**